specification/schema/tileset.schema.json

```diff
@@ -19,8 +19,54 @@
             "description": "A dictionary object of metadata about per-feature properties.",
             "additionalProperties": {
                 "$ref": "properties.schema.json"
+            },
+            "deprecated": true
+        },
+        "schema": {
+            "allOf": [
+                {
+                    "$ref": "schema.schema.json"
+                }
+            ],
+            "description": "An object defining classes and enums."
+        },
+        "schemaUri": {
+            "type": "string",
+            "description": "The URI (or IRI) of the external schema file.",
+            "format": "iri-reference"
+        },
+        "statistics": {
+            "allOf": [
+                {
+                    "$ref": "statistics.schema.json"
+                }
+            ],
+            "description": "An object containing statistics about entities."
+        },
+        "groups": {
+            "type": "object",
+            "description": "A dictionary, where each key is a group ID and each value is an object defining the group. Group IDs may contain only alphanumeric and underscore characters",
+            "minProperties": 1,
+            "additionalProperties": {
+                "$ref": "group.schema.json"
             }
         },
+        "metadata": {
```

Comment:
Most of the properties in `3DTILES_metadata` have been hoisted up to root scope. There are a few reasons for doing that:

* Schemas are so central to 3D Tiles 1.1 that it feels right to make it a top level property
* `statistics` is a replacement for `properties` which was also at the top level
* `groups` are really a distinct concept from metadata and don't belong under a metadata sub-object. Arguably `3DTILES_metadata` should not have groups; that should be a separate extension.
* Tileset, tile, and content all have a `metadata` property that is a `metadataEntity.schema.json`. The symmetry is nice.